submissions/alstep07/html-css-popup/index.html

```diff
@@ -0,0 +1,127 @@
+							<div class="menu__item-descr">Contacts</div>
+						</a>
+						<a href="#" class="menu__item">
+							<img class="menu__item-img" src="./img/tanslation.png" alt="translate">
+							<div class="menu__item-descr">Translate</div>
+						</a>
+						<a href="#" class="menu__item">
+							<img class="menu__item-img" src="./img/videos.png" alt="videos">
+							<div class="menu__item-descr">Videos</div>
+						</a>
+						<a href="#" class="menu__item">
+							<img class="menu__item-img" src="./img/search.png" alt="search">
+							<div class="menu__item-descr">Search</div>
+						</a>
+						<a href="#" class="menu__item">
+							<img class="menu__item-img" src="./img/photos.png" alt="photo">
+							<div class="menu__item-descr">Photo</div>
+						</a>
+						<a href="#" class="menu__item">
+							<img class="menu__item-img" src="./img/apps.png" alt="apps">
+							<div class="menu__item-descr">Apps</div>
+						</a>
+						<a href="#" class="menu__item">
+							<img class="menu__item-img" src="./img/documents.png" alt="document">
+							<div class="menu__item-descr">Document</div>
+						</a>
+						<input id="menu-extra" class="menu__extra-input" type="checkbox">
+						<label class="menu__extra-btn" for="menu-extra">More</label>
+						<div class="menu__extra">
+							<a href="#" class="menu__item">
+								<img class="menu__item-img" src="./img/chat.png" alt="chat">
+								<div class="menu__item-descr">Chat</div>
+							</a>
+							<a href="#" class="menu__item">
+								<img class="menu__item-img" src="./img/classroom.png" alt="classroom">
+								<div class="menu__item-descr">Classroom</div>
+							</a>
+							<a href="#" class="menu__item">
+								<img class="menu__item-img" src="./img/collection.png" alt="collection">
+								<div class="menu__item-descr">Collection</div>
+							</a>
+							<a href="#" class="menu__item">
+								<img class="menu__item-img" src="./img/mars.png" alt="mars">
+								<div class="menu__item-descr">Mars</div>
+							</a>
+							<a href="#" class="menu__item">
+								<img class="menu__item-img" src="./img/blogs.png" alt="blogs">
+								<div class="menu__item-descr">Blogs</div>
+							</a>
+							<a href="#" class="menu__item">
+								<img class="menu__item-img" src="./img/promo.png" alt="promos">
+								<div class="menu__item-descr">Promos</div>
+							</a>
+							<a href="https://github.com/kottans/frontend" 
+							   class="menu__extra-link">
+							   Even more from Kottans
+							</a>
+						</div>
+					</div>
+				</div>
+			</li>
+			<li class="nav__elem">
+				<a class="nav__btn" href="#">
+					<img class="bell__img" src="./img/bell.png" alt="bell">
+				</a>
+			</li>
+			<li class="nav__elem">
+				<a class="nav__btn" href="https://github.com/alstep07/kottans-frontend">
+					<img class="user__img" src="./img/user_photo.png" alt="user_photo">
+				</a>
+			</li>
+		</ul>
+	</nav>
+</body>
+
+</html>
```

Comment:
could you please add an empty line at the end of the file?  if you see this symbol - it means that the empty line is needed   
![empty line](https://user-images.githubusercontent.com/10871447/100937228-5d332280-34fb-11eb-8c85-b940aa2dee45.png)

Reply:
added